src/print.js

```diff
@@ -0,0 +1,3 @@
+// export default function printMe() {
+//   console.log('I get called from print.js!');
```

Comment:
- [ ] kindly follow the best JavaScript, Do not commit console.log to your repo. It’s ugly, it kills performance and it can make confidential data visible to anyone using the browser tools to look at your website.